### PR TITLE
Use absolutePath in fsstorage class

### PR DESCRIPTION
**Pull request by patriotyk**

### CODE REVIEW

File: src/utils.cc [hunk 1]
```diff
@@ -322,6 +322,14 @@ int Utils::shell(const std::string &command, std::string *output) {
   return WEXITSTATUS(exitcode);
 }
 
+boost::filesystem::path Utils::absolutePath(const boost::filesystem::path &root, const boost::filesystem::path &file) {
+  if (file.is_absolute() || root.empty()) {
+    return root;
```
Comment (patrickvacek) [inline src/utils.cc]: I think you mean `return file`!

**Comment by patrickvacek** [comment]
I think this might be slightly backwards -- I think fsstorage is probably fine as it is, but there are plenty of places with the Config and SecondaryConfig classes where this is more of an issue, like the `StorageConfig.path` along with the `uptane_metadata_path`, etc. This might make the `ImportConfig` redundant, unless that has another use that I don't know of.

File: src/utils.cc (hunk 1 above)
Comment (patriotyk) [reply src/utils.cc]: Yes, You are right.

**Comment by patriotyk** [comment]
I think You can merge this. All places where we concatenate path with uptane_metadata_path have been replaced with helper function.

**Pull request merged**